src/runtime_src/core/pcie/driver/linux/xocl/subdev/xmc.c

```diff
@@ -944,7 +1017,7 @@ static int get_temp_by_m_tag(struct xocl_xmc *xmc, char *m_tag)
 		return -ENODEV;
 
 	if (!strncmp(m_tag, "HBM", 3)) {
-		xmc_sensor(xmc->pdev, HBM_TEMP, &ret, SENSOR_INS);
+		xmc_sensor(xmc->pdev, HBM_TEMP1, &ret, SENSOR_INS);
```

Comment:
All hbm banks has one sensor? then what is the usage for HBM_TEMP2?

Reply:
My understanding of it is that the second sensor reading is used internally by the SC to calculate the temperature and by the end-user to check correct operation of the board.

Reply:
I'm not following... if it's used internally by the SC, then please don't expose it. If it's used by end-user, then what is the difference b/w hbm-tmp1 and hbm-tmp2? the hbm-tmp1 is also used by end-user.